dockers/relaydc/docker-compose.yml

```diff
@@ -24,15 +24,15 @@ services:
     restart: always
     ports: [ '8080:80' ] #bypass https-portal
     # add line for each relay
-    command: 
```

Comment:
random space change..

Reply:
That's probably IDEA removing redundant spacing that shouldn't have been there in the first place..